### PR TITLE
[BD-3494] Update Live Activities FAQ

### PR DESCRIPTION
**Pull request by jerielng**
### Summary
Updates some of the Live Activities FAQs with some of the more recent questions we've received.

### Related PRs, issues, or features (optional)

- [BD-3494](https://jira.braze.com/browse/BD-3494)

### Feature release date (optional)

- N/A

### Contributor checklist

- [x] I confirm that my PR meets the following:
    - I signed the [Contribution License Agreement (CLA)](https://www.braze.com/docs/cla).
    - My style and voice follows the [Braze Style Guide](https://docs.google.com/document/u/2/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub).
    - My content contains correct spelling and grammar.
    - All links are working correctly.
    - If I renamed or moved a file or directory, I set up [URL redirects](https://www.braze.com/docs/contributing/content_management/redirecting_urls) for each file.
    - If I updated or replaced an image, I did not remove the original image file from the repository. (For more information, see [Updating an image](https://www.braze.com/docs/contributing/content_management/images/#updating-an-image)).
    - If my PR is related to a paid SKU, third party, SMS, AI, or privacy, I have received written approval from Braze Legal.

### Submitting for review

If your PR meets the above requirements, select **Ready for review**, then add a reviewer:
- **Non-Braze contributors:** Add `braze-inc/docs-team` as the reviewer.
- **Braze employees:** Have any relevant subject matter experts (like engineers or product managers) review your work first, then add the [tech writer assigned to your specific vertical](https://confluence.braze.com/pages/viewpage.action?pageId=75039896). If you're not sure which tech writer to add, you can add `braze-inc/docs-team` instead. 

Thanks for contributing! We look forward to reading your work.

### CODE REVIEW

**Review by vanessaland — APPROVED** [review]

File: _docs/_developer_guide/platform_integration_guides/swift/live_activities/faq.md [hunk 1]
```diff
@@ -38,20 +38,26 @@ Live Activities and push notifications occupy different screen real estate and w
...
 
-Apple requires that the user initiates the Live Activity through some action in your app, for example, by placing an order. This Live Activity token expires after eight hours. 
+Live Activity update tokens expire after eight hours.
 
 ### Do Live Activities require push primers?
 
-[Push primers]({{site.baseurl}}/user_guide/message_building_by_channel/push/best_practices/push_primer_messages/) are a best practice to prompt your users to opt in to push notifications from your app. However, there is no system prompt to opt into Live Activities. Users are, by default, opted into Live Activities when they upgrade to iOS 16.1+.
+[Push primers]({{site.baseurl}}/user_guide/message_building_by_channel/push/best_practices/push_primer_messages/) are a best practice to prompt your users to opt in to push notifications from your app. However, there is no system prompt to opt into Live Activities. Users are, by default, opted into Live Activities for any individual app when they install that app on iOS 16.1+. This permission can be disabled or re-enabled in the device settings on a per-app basis.
```
Comment (hokstuff) [inline _docs/_developer_guide/platform_integration_guides/swift/live_activities/faq.md:47]: Ah I didn't realize this - so this is basically the "provisional" behavior comes by default for Live Activities?

**Review by hokstuff — APPROVED** [review]
LGTM!

File: _docs/_developer_guide/platform_integration_guides/swift/live_activities/faq.md (hunk 1 above)
Comment (jerielng) [reply _docs/_developer_guide/platform_integration_guides/swift/live_activities/faq.md:47]: Yea basically if I install an app that has implemented Live Activities, the feature will be turned on by default. If I don't want to receive any Live Activities, I'll need to go into that app's settings on my phone and disable it.

**Comment by jerielng** [comment]
Hey @internetisaiah this should be ready for a tech writer review now. Thanks!

File: _docs/_developer_guide/platform_integration_guides/swift/live_activities/faq.md (hunk 1 above)
Comment (internetisaiah) [inline _docs/_developer_guide/platform_integration_guides/swift/live_activities/faq.md:47]:
```suggestion
[Push primers]({{site.baseurl}}/user_guide/message_building_by_channel/push/best_practices/push_primer_messages/) are a best practice to prompt your users to opt in to push notifications from your app. However, there is no system prompt to opt into Live Activities. By default, users are opted into Live Activities for an individual app when the user installs that app on iOS 16.1 or later. This permission can be disabled or re-enabled in the device settings on a per-app basis.
```

File: _docs/_developer_guide/platform_integration_guides/swift/live_activities/faq.md [hunk 2]
```diff
@@ -38,20 +38,26 @@ Live Activities and push notifications occupy different screen real estate and w
...
 
 ## Technical topics and troubleshooting
 
 ### How do I know if Live Activities has errors?
 
 Any Live Activity errors will be logged in the Braze dashboard in the [Message Activity Log]({{site.baseurl}}/user_guide/administrative/app_settings/message_activity_log_tab/), where you can filter by "LiveActivity Errors".
 
+### After sending a push-to-start notification, why haven't I received my Live Activity?
+
+First, verify that your payload includes all the required fields described in the [`messages/live_activity/start`]({{site.baseurl}}/api/endpoints/messaging/live_activity/start) endpoint and that the `activity_attributes` and `content_state` fields match the properties defined in your project code. If you are certain that the payload is correct, you may be rate limited by APNs. This limit is imposed by Apple and not by Braze.
```
Comment (internetisaiah) [inline _docs/_developer_guide/platform_integration_guides/swift/live_activities/faq.md:57]:
```suggestion
First, verify that your payload includes all the required fields described in the [`messages/live_activity/start`]({{site.baseurl}}/api/endpoints/messaging/live_activity/start) endpoint. The `activity_attributes` and `content_state` fields should match the properties defined in your project's code. If you're certain that the payload is correct, its possible you may be rate-limited by APNs. This limit is imposed by Apple and not by Braze.
```

File: _docs/_developer_guide/platform_integration_guides/swift/live_activities/faq.md [hunk 3]
```diff
@@ -38,20 +38,26 @@ Live Activities and push notifications occupy different screen real estate and w
...
+
+To verify that your push-to-start notification successfully arrived at the device but was not displayed due to rate limits, you can debug your project using the Console app on your Mac. Attach the recording process for your desired device, and in the search bar, filter logs by `process:liveactivitiesd`.
```
Comment (internetisaiah) [inline _docs/_developer_guide/platform_integration_guides/swift/live_activities/faq.md:59]:
```suggestion
To verify that your push-to-start notification successfully arrived at the device but was not displayed due to rate limits, you can debug your project using the Console app on your Mac. Attach the recording process for your desired device, then filter the logs by `process:liveactivitiesd` in the search bar.
```

**Review by internetisaiah — APPROVED** [review]
LGTM

**Pull request merged**